spec/models/manageiq/providers/kubernetes/container_manager/refresh_parser_spec.rb

```diff
@@ -525,6 +525,47 @@
     end
   end
 
+  describe "cross_link_node" do
+    before :each do
+      @node = RecursiveOpenStruct.new(
+        :identity_system => "f0c1fe7e-9c09-11e5-bb22-28d2447dcefe",
+        :identity_infra  => "aws:///us-west-1/aws-id"
```

Comment:
@enoodle this should be empty, you are assigning a value on some of the tests anyways.